services/streams/streams_queue/main.go

```diff
@@ -0,0 +1,21 @@
+package streams_queue
+
+import (
+	"github.com/leanspace/terraform-provider-leanspace/provider"
+)
+
+var StreamQueueDataType = provider.DataSourceType[Stream, *Stream]{
+	ResourceIdentifier: "leanspace_streams_queue",
```

Comment:
Shouldn't the name be rather stream_queues?
```suggestion
	ResourceIdentifier: "leanspace_stream_queues",
```